src/Magnum/DartIntegration/Test/DartSkeletonTest.cpp

```diff
@@ -0,0 +1,267 @@
+    properties.mT_ParentBodyToJoint.translation() = Eigen::Vector3d(0, 0, default_height);
+    properties.mRestPositions[0] = default_rest_position;
+    properties.mSpringStiffnesses[0] = default_stiffness;
+    properties.mDampingCoefficients[0] = default_damping;
+
+    // Create a new BodyNode, attached to its parent by a RevoluteJoint
+    dart::dynamics::BodyNodePtr bn = pendulum->createJointAndBodyNodePair<dart::dynamics::RevoluteJoint>(
+                                                 parent, properties, dart::dynamics::BodyNode::AspectProperties(name))
+                                         .second;
+
+    // Make a shape for the Joint
+    const double R = default_width / 2.0;
+    const double h = default_depth;
+    std::shared_ptr<dart::dynamics::CylinderShape> cyl(new dart::dynamics::CylinderShape(R, h));
+
+    // Line up the cylinder with the Joint axis
+    Eigen::Isometry3d tf(Eigen::Isometry3d::Identity());
+    tf.linear() = dart::math::eulerXYZToMatrix(
+        Eigen::Vector3d(90.0 * M_PI / 180.0, 0, 0));
+
+    auto shapeNode = bn->createShapeNodeWith<dart::dynamics::VisualAspect>(cyl);
+    shapeNode->getVisualAspect()->setColor(dart::Color::Blue());
+    shapeNode->setRelativeTransform(tf);
+
+    // Set the geometry of the Body
+    setGeometry(bn);
+
+    return bn;
+}
+
+namespace Magnum { namespace DartIntegration { namespace Test {
+
+namespace {
+
+    using namespace Math::Literals;
+
+    typedef SceneGraph::Object<SceneGraph::MatrixTransformation3D> Object3D;
+    typedef SceneGraph::Scene<SceneGraph::MatrixTransformation3D> Scene3D;
+
+}
+
+struct DartSkeletonTest : TestSuite::Tester {
+    explicit DartSkeletonTest();
+
+    void test();
+    void test_urdf();
+    void test_texture();
+};
+
+DartSkeletonTest::DartSkeletonTest() {
+    addTests({&DartSkeletonTest::test});
+    addTests({&DartSkeletonTest::test_urdf});
+    addTests({&DartSkeletonTest::test_texture});
+}
+
+void DartSkeletonTest::test() {
+    // Create an empty Skeleton with the name "pendulum"
+    dart::dynamics::SkeletonPtr pendulum = dart::dynamics::Skeleton::create("pendulum");
+
+    // Add each body to the last BodyNode in the pendulum
+    dart::dynamics::BodyNode* bn = makeRootBody(pendulum, "body1");
+    bn = addBody(pendulum, bn, "body2");
+    bn = addBody(pendulum, bn, "body3");
+    bn = addBody(pendulum, bn, "body4");
+    bn = addBody(pendulum, bn, "body5");
+
+    // Create a world and add the pendulum to the world
+    dart::simulation::WorldPtr world(new dart::simulation::World);
+    world->addSkeleton(pendulum);
+
+    Scene3D scene;
+    Object3D* obj = new Object3D{&scene};
+
+    DartSkeleton* skel = new DartSkeleton{obj, pendulum};
+
+    // Set the initial joint positions so that the pendulum
+    // starts to swing right away
+    pendulum->getDof(1)->setPosition(120. * M_PI / 180.0);
+    pendulum->getDof(2)->setPosition(20. * M_PI / 180.0);
+    pendulum->getDof(3)->setPosition(-50. * M_PI / 180.0);
```

Comment:
There's the `Constants::pi()` constant if you want, you could also do `Float(Rad(120.0_degf))`, but that might look scary and I'm not forcing you to do that ;)